cloud-aws/src/main/resources/templates/aws-cf-dbstack.ftl

```diff
@@ -46,7 +46,7 @@
     },
     "EngineVersionParameter": {
         "Type": "String",
-        "Default": "10.14",
+        "Default": "10",
```

Comment:
I think the default can be deleted. The major version parameters is always supplied.

Reply:
"Default": "10" this should be removed